### ISSUE TITLE
Some issues I noticed with docker after update

### ISSUE THREAD

**Issue by dakotabenjamin** [developmentseed/osm-seed#183]
Been playing around with osm-seed to get it working with docker. I had to make some changes to make it work, so I will document them here for you if that helps improve the docker implementation. 
1. The docker-compose.yml was not updated for the new folder structure. i.e. 
```
-      context: ./db-backup-restore
+      context: ./images/backup-restore
```
2. For the individual Dockerfiles, I had to combine the `apt-get update` and `apt-get install ...` lines into one because of docker's caching: 
```
-RUN apt-get update
-RUN apt-get install -y \
+RUN apt-get update && apt-get install -y \
```
This was done on `backup-restore`, `db`, `osm-processor`, `tiler-db`, `tiler-imposm`, and `tiler-server`.

I am also still struggling to get iD to work within `web`. As suggested by @geohacker and @batpad I changed the reference in the `start.sh` file from `$workdir/config/application.yml` to `$workdir/config/settings.local.yml` and added the `OSM_id_key` to my `.env` file. The error I get still looks like this: 
![image](https://user-images.githubusercontent.com/1847818/108556984-d99cfc80-72c5-11eb-96ae-578d06cf2d8a.png)

The file is being saved correctly to the docker container, as checked by `docker exec` into the container. No other issues in the console log or network tab. Any clues here?

**Comment by geohacker** [comment]
Thanks for the ticket @dakotabenjamin! Sorry that iD is still not working. 
@Rub21 do you think you can take a look at why iD isn't working as part of the website?

**Comment by dakotabenjamin** [comment]
Happy to give either of you access to the osmseed.hotosm.org server if you wanted to take a look. DM me on HOT Slack if you want that.

**Comment by dakotabenjamin** [comment]
Update: We tried using some older commits of openstreetmap-website but these did not work. Perhaps we could not find the "right" one but we tried `2fd0d2304f05bdac456e654681cd26565c2d7f08` (from this repos `master`) and another from October 2020.

**Comment by Rub21** [comment]
@dakotabenjamin @geohacker , looks the error is because of the lates version,  I have  not tried yet running the containers since the last refactoring.  but i think a solution can be changing the version of the Api.

 - https://github.com/developmentseed/osm-seed/blob/develop/images/web/Dockerfile#L112-L115
  replace with this one:

```
# Clone the openstreetmap-website
RUN rm -rf $workdir
RUN git clone https://github.com/openstreetmap/openstreetmap-website.git $workdir 
WORKDIR $workdir
RUN git checkout 39f0b501e8cefea961447de9c8076e20fa3adbb4
```

**Comment by dakotabenjamin** [comment]
Thanks for looking into this. with that commit I ran into some compatible dependency issues: 
```
Step 44/71 : RUN bundle install
 ---> Running in 3ce6d5d4e8cc
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and
installing your bundle as root will break this application for all non-root
users on this machine.
Fetching gem metadata from https://rubygems.org/........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "libv8":
  In snapshot (Gemfile.lock):
    libv8 (= 3.16.14.19)

  In Gemfile:
    mini_racer was resolved to 0.1.0.beta.3, which depends on
      libv8 (= 5.0.71.48.1beta2)

    jshint was resolved to 1.5.0, which depends on
      therubyracer (~> 0.12.1) was resolved to 0.12.3, which depends on
        libv8 (~> 3.16.14.15)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
ERROR: Service 'web' failed to build: The command '/bin/sh -c bundle install' returned a non-zero code: 6
```
I tried to resolve with different version specs for mini_racer or libv8 but that didn't work. I also tried installing 'therubyracer' instead of mini_racer, but no avail. Then I tried to use the Dockerfile code from before the folder structure changes. Still other version dependency issues such as nokogiri and zeitwerk requiring newer versions of ruby. 

Am I looking in the right direction here? What should I try next?
